arb.h

```diff
@@ -1007,6 +1007,14 @@ _arb_vec_estimate_allocated_bytes(slong len, slong prec)
     return size;
 }
 
+int arb_load_str(arb_t res, const char * data);
```

Comment:
@fredrik-johansson @videlec @isuruf: So, what is the final verdict here? Is `x_load/dump_str/file` a good naming? There does not seem to be a clear standard really.

Reply:
Its fine in my opinion.